docs/votervoice.apib

```diff
@@ -1245,6 +1245,75 @@ Patch an existing contact record contactId using the temporaryContactId.
 
 + Response 204 (application/json)
 
+
+## Contacts Field Mappings [/contacts/mappings{?association,remoteSource,direction}]
+The contact field mappings resource can be used to search the collection of configured contact field mappings.
+
+### Get Contact Field Mappings [GET]
+This resource takes in the association site information and returns contacts field mappings for the association.
+
++ Request
+    + Headers
+    ```
+        Authorization: YOUR_API_KEY
+        Content-Type: application/json
+    ```
+    
++ Parameters
+    + association: `YOUR_ASSOCIATION_IDNAME` (string, required) - the VoterVoice acronym for your association.
+    + remoteSource: `Upload Tool` (string, optional) - Expected values are "Upload Tool", "Salesforce", or "FiscalNote".
```

Comment:
Is "FiscalNote" an internal type so to speak? i.e. would anyone outside of us use it?

Reply:
Probably not, but I didn't see the harm in putting it.